docs/cody/proxy-setup.mdx

```diff
@@ -0,0 +1,112 @@
+
+# Proxy Setup Guide
+
+## Introduction
+This guide will walk you through setting up HTTP, HTTPS, and SOCKS proxies in Visual Studio Code (VS Code) and JetBrains IDEs. It also includes instructions for handling self-signed certificates on macOS and Windows.
+
+## Proxy Setup for VS Code
+
+### Accessing the Proxy Settings
+1. Open VS Code.
+2. Navigate to the settings window of VS Code.
```

Comment:
@arafatkatze Just checking here that I can't achieve the same effect without editing the Settings.JSON file? Does it work if I just input the variables into the text box on the UI directly?

Reply:
It could work for http proxy but for https proxy you might need to set extra stuff which is why I recommended the standard go to settings.json